indra/databases/pubchem_client.py

```diff
@@ -74,3 +75,28 @@ def get_preferred_compound_ids(pubchem_cid):
                 pref_ids |= set(pref_cpd['Value']['Number'])
     pref_ids = sorted([str(pid) for pid in pref_ids])
     return pref_ids
+
+
+def get_pmids(pubchem_cid: str) -> List[str]:
+    """Return xref PMIDs for a given PubChem CID.
+
+    Parameters
+    ----------
+    pubchem_cid : str
+        The PubChem CID whose xref PubMedID's will be returned.
+
+    Returns
+    -------
+    list
```

Comment:
Documentation of type is not complete - it's actually better to just have it as actual type hints
```suggestion
    :
```

Reply:
I just fixed it. It will now return a list of 'str'